### ISSUE TITLE
Blank browser screen on initial app launch in new browser

### ISSUE THREAD

**Issue by basilv** [bcgov/nr-forests-access-management#386]
Describe the Bug
When I launch the app by opening a new browser (e.g. after clearing history) and type in URL, I get a blank page. The web console shows the following error:
Uncaught TypeError: Cannot read properties of null (reading 'fam_cognito_region')

Expected Behaviour
The application screen displays properly. This does happen when I refresh the page.

Actual Behaviour
Blank screen is displayed. In the code, it appears it is initializing main.ts and specifically awsExports before env.js is called to initialize the environment. Perhaps move initializing of env.js into main.ts???

Additional context
Observed in Edge and Chrome.

**Comment by ianliuwk1019** [comment]
PR is ready.
In summary:
From observations, this initial blank page only happens during user's "**first time**" (particularly when no browser caching from previous loading) accessing the FAM site.
Found that web browser stack can use "**preload**" for preloading some content for performance and that seems to solve the problem. Tested few times on browsers (Chrome/Edge) with clear caching and inPrivate mode, I observed it is consistent now with no blank page issue.
Hopefully it does solve the problem.

![image.png](https://images.zenhubusercontent.com/62a9ff93d8506f14758ef03a/5ef17d5e-a5b8-458b-af37-9bf9b03118ee)
As can be seen above, **"env.json"** is preloaded and ready first, and main **"js"** code is then queued and loaded later.
